### PR TITLE
Add `speech/recognize` capability

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds `speech/recognize` profile with ability to convert speech to text.

Providers:
- google-apis-speech
- ibm-cloud-speech-to-text
- azure-speech-to-text
- mock

Audio input file is Base64 encoded text. I did not find better option, passing Buffer as input is not supported.

Currently only `wav` audio format is supported. ~~It is not currently possible to support other audio formats by providers that accept `audio/*` content types, because one-sdk do not support dynamic content type changes.~~

- [x] update @superfaceai/one-sdk version to 1.0.1 before merging

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "@superfaceai/ast": "^1.0.0",
     "@superfaceai/cli": "^1.0.1",
-    "@superfaceai/one-sdk": "^1.0.0",
+    "@superfaceai/one-sdk": "^1.0.1-rc.1",
```
Comment (Jakub-Vacek) [inline package.json:26]: Not sure if we want to have RCs in main. @freaz what do you think?

File: capabilities/speech/recognize/profile.supr [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+"""
+
+name = "speech/recognize"
+version = "1.0.0"
+
+"""
+Speech recognition
+Real-time speech recognition.
+"""
+usecase Recognize safe {
```
Comment (Jakub-Vacek) [inline capabilities/speech/recognize/profile.supr:13]: Maybe add examples?

File: package.json (hunk 1 above)
Comment (janhalama) [reply package.json:26]: I want to update to 1.0.1 but did not want to block review with it. There is task for it in description.

File: capabilities/speech/recognize/profile.supr (hunk 2 above)
Comment (janhalama) [reply capabilities/speech/recognize/profile.supr:13]: examples added

**Review by freaz — APPROVED** [review]
Looks good to me.

Only thing I stopped on was `maxAlternatives` which seems to be provider specific.

File: .env.capabilities.example [hunk 3]
```diff
@@ -41,8 +43,11 @@ COMMUNICATION_SENDMESSAGE_TO=+xxx
 
 # PARAMETERS
 
+AZURE_SPEECH_TO_TEXT_REGION=xxx
 IBM_CLOUD_NATURAL_LANGUAGE_UNDERSTANDING_INSTANCE_ID=xxx
 IBM_CLOUD_NATURAL_LANGUAGE_UNDERSTANDING_REGION=xxx
+IBM_CLOUD_SPEECH_TO_TEXT_INSTANCE_ID=xxx
+IBM_CLOUD_SPEECH_TO_TEXT_REGION=xxx
 IBM_CLOUD_TEXT_TO_SPEECH_INSTANCE_ID=xxx
 IBM_CLOUD_TEXT_TO_SPEECH_REGION=xxx
```
[the remark this answers is not in the document]
Comment (janhalama) [reply .env.capabilities.example:52]: We can have only one `IBM_CLOUD_REGION` env variable in station. `IBM_CLOUD_..._INSTANCE_ID` is specific for each IBM cloud API.

**Comment by janhalama** [comment]
> Only thing I stopped on was maxAlternatives which seems to be provider specific.

You are right, not all providers supports alternatives. 

Providers that returns just one alternative will still comply with profile definition. Profile`maxAlternatives` description: _Maximum number of recognition hypotheses to be returned. The server may return fewer than maxAlternatives._

File: package.json (hunk 1 above)
Comment (janhalama) [reply package.json]: one-sdk updated to v1.0.1

**Pull request merged**